sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -177,6 +177,109 @@ def begin_recognize_receipts_from_url(self, receipt_url, **kwargs):
             **kwargs
         )
 
+    @distributed_trace
+    def begin_recognize_business_cards(
+            self,
+            business_card,
+            **kwargs
+    ):
+        # type: (Union[bytes, IO[bytes]], Any) -> LROPoller[List[RecognizedForm]]
+        """Extract field text and semantic values from a given business card.
+        The input document must be of one of the supported content types - 'application/pdf',
+        'image/jpeg', 'image/png' or 'image/tiff'.
+
+        See fields found on a business card here:
+        https://aka.ms/formrecognizer/businesscardfields
+
+        :param business_card: JPEG, PNG, PDF and TIFF type file stream or bytes.
+        :type business_card: bytes or IO[bytes]
+        :keyword str locale: Locale of the business card. Default locale is en-US. Other supported locales
```

Comment:
I think it will be better to avoid saying what the default is, in case the service decides a different value.

Reply:
That's fair, I'll remove the default from the docstring